### PR TITLE
Standardize open source mentorships

### PR DESCRIPTION
**Pull request by bkeepers**
This pull request updates the structure of this repo to somewhat standardize how we participate in open source mentorship programs.
- [x] Rename this repo to github/mentorships
- [x] Add labels: `question`, `project` and `program`
- [x] Update repository description
- [ ] Open issues for projects we have lined up
- [ ] Wait to hear back about applications to mentorship programs  :pray: :8ball:
- [ ] Open issues detailing our involvement in each program
- [x] :shipit: 

/cc @johndbritton @mikemcquaid @mlinksva @tberman

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+>
+> Mentors: [@kevinsawicki](https://github.com/kevinsawicki)
+
+Mentoring is incredibly rewarding, and providing a positive experience for both you and the student is an big commitment that is not to be taken lightly. Please talk to your team before volunteering.
+
+Mentoring Resources:
+
+- [Outreachy's info for Mentors](https://wiki.gnome.org/Outreachy/Admin/InfoForMentors)
+- [Google Summer of Code Mentor's Guide](http://en.flossmanuals.net/GSoCMentoring/what-is-gsoc/)
+
```
Comment (johndbritton) [inline CONTRIBUTING.md:49]: RGSoC -- http://railsgirlssummerofcode.org/guide/projects/

File: README.md [hunk 2]
```diff
@@ -1,200 +1,9 @@
-# Google Summer of Code
+# GitHub Open Source Mentorships
```
Comment (johndbritton) [inline README.md:1]: :heart:

**Comment by johndbritton** [comment]
This looks great, I love that we're able to participate in multiple programs and standardize the process.

**Comment by MikeMcQuaid** [comment]
Makes sense to me :+1:

**Comment by bkeepers** [comment]
Awesome. I'm going to go ahead and merge this. The remaining checkboxes will get checked off in the coming weeks.

**Pull request merged**